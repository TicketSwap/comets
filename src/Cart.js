import React from 'react'
import { Icon } from './Icon'

export const Cart = props => (
  <Icon a11yTitle="Cart" {...props}>
    {({ isSmallIcon }) => {
      if (isSmallIcon) {
        return (
          <path d="M9.5 17C10.3284 17 11 17.6716 11 18.5C11 19.3284 10.3284 20 9.5 20C8.67157 20 8 19.3284 8 18.5C8 17.6716 8.67157 17 9.5 17ZM16.5 17C17.3284 17 18 17.6716 18 18.5C18 19.3284 17.3284 20 16.5 20C15.6716 20 15 19.3284 15 18.5C15 17.6716 15.6716 17 16.5 17ZM4.84577 4C5.5051 4 6.11824 4.32458 6.48974 4.86095L6.57817 5.00061C6.93553 5.61907 7.59559 6 8.30987 6H19.7802C20.3325 6 20.7802 6.44772 20.7802 7C20.7802 7.06585 20.7737 7.13154 20.7608 7.19612L19.1608 15.1961C19.0673 15.6635 18.6569 16 18.1802 16H7.8198C7.34312 16 6.93271 15.6635 6.83922 15.1961L5.16078 6.80388C5.06729 6.33646 4.65688 6 4.1802 6H3C2.44772 6 2 5.55228 2 5C2 4.44772 2.44772 4 3 4H4.84577Z" />
        )
      }

      return (
        <path d="M7.5 8H28a1.001 1.001 0 0 1 .965 1.263l-3 11A1 1 0 0 1 25 21H9a1 1 0 0 1-.978-.79L5.19 7H2a1 1 0 1 1 0-2h4a1 1 0 0 1 .978.79L7.5 8zM9 25a2 2 0 1 1 3.999-.001 2 2 0 0 1-4 .001zm12 0a2 2 0 1 1 3.999-.001 2 2 0 0 1-4 .001z" />
      )
    }}
  </Icon>
)
