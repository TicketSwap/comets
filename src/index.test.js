import React from 'react'
import { render } from '@testing-library/react'
import { Cart } from './Cart'

describe('Icon', () => {
  it('matches snapshot', () => {
    const { container } = render(<Cart />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
