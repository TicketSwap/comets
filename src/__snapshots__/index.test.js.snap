// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon matches snapshot 1`] = `
<span
  class="icon css-yj93w3-SvgWrapper elpsssu1"
>
  <svg
    aria-label="Cart"
    class="css-1dx98cs-InlineSvg elpsssu0"
    clip-rule="evenodd"
    fill-rule="evenodd"
    focusable="false"
    preserveAspectRatio="xMidYMid meet"
    role="presentation"
    stroke-linejoin="round"
    stroke-miterlimit="1.414"
    viewBox="0 0 32 32"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M7.5 8H28a1.001 1.001 0 0 1 .965 1.263l-3 11A1 1 0 0 1 25 21H9a1 1 0 0 1-.978-.79L5.19 7H2a1 1 0 1 1 0-2h4a1 1 0 0 1 .978.79L7.5 8zM9 25a2 2 0 1 1 3.999-.001 2 2 0 0 1-4 .001zm12 0a2 2 0 1 1 3.999-.001 2 2 0 0 1-4 .001z"
    />
  </svg>
</span>
`;
